migrator/migrator.go

```diff
@@ -386,7 +388,8 @@ func (m Migrator) MigrateColumn(value interface{}, field *schema.Field, columnTy
 			// Since the following code is frequently called in the for loop, reg optimization is needed here
 			matches := regRealDataType.FindAllStringSubmatch(realDataType, -1)
 			matches2 := regFullDataType.FindAllStringSubmatch(fullDataType, -1)
-			if (len(matches) == 1 && matches[0][1] != fmt.Sprint(field.Size) || !field.PrimaryKey) && (len(matches2) == 1 && matches2[0][1] != fmt.Sprint(length)) {
+			if (len(matches) == 1 && matches[0][1] != strconv.Itoa(field.Size) || !field.PrimaryKey) &&
+				(len(matches2) == 1 && matches2[0][1] != strconv.FormatInt(length, 10)) {
```

Comment:
Please revert this line, I will accept this PR. 

`strconv.Itoa` and `strconv.FormatInt` looks like more complex, not clearly then `fmt.Sprint`.

Reply:
@huacnlee ok,i has reverted this line.